tests/test_dimension.py

```diff
@@ -1173,21 +1165,16 @@ def test_topofusion_w_subdims_conddims(self):
 
         # Check generated code -- expect the gsave equation to be scheduled together
         # in the same loop nest with the fsave equation
-        assert len(op._func_table) == 3
 
-        exprs = FindNodes(Expression).visit(op._func_table['bf0'].root)
-        assert len(exprs) == 2
-        assert exprs[0].write is f
-        assert exprs[1].write is g
+        exprs = FindNodes(Expression).visit(op)
```

Comment:
same as above... it's not clear anymore that we have three separate "blocks" , which the bfs were implicitly denoting